RLTest/utils.py

```diff
@@ -151,6 +150,8 @@ def fix_modulesArgs(modules, modulesArgs, defaultArgs=None, haveSeqs=True):
         n = len(modules) - num_mods
 
     if n > 0:
+        if not modulesArgs:
```

Comment:
Because we allow empty `modulesArgs` (for instance, if `modules` are empty). We only pad if there's a good reason (like, `modules` or `defaultArgs` being of a larger size).